DISCLAIMER-WIP

```diff
@@ -0,0 +1,23 @@
+Apache XTable (incubating) is an effort undergoing incubation at The Apache
+Software Foundation (ASF), sponsored by the Apache Incubator.
+Incubation is required of all newly accepted projects until a further review
+indicates that the infrastructure, communications, and decision making process
+have stabilized in a manner consistent with other successful ASF projects.
+While incubation status is not necessarily a reflection of the completeness or
+stability of the code, it does indicate that the project has yet to be fully
+endorsed by the ASF.
+
+Some of the incubating project’s releases may not be fully compliant with ASF
+policy. For example, releases may have incomplete or un-reviewed licensing
+conditions. What follows is a list of known issues the project is currently
+aware of (note that this list, by definition, is likely to be incomplete):
+
+  * There were few known vulnerabilities in the codebase which have been fixed.
```

Comment:
I'm not sure this is required since these issues have already been addressed in the WIP release? Regarding licenses, we did a review during the donation, and probably not much has changed, but we'll need to make sure we are compliant. If we don't have a list of issues we are aware of, we could leave it empty.

Reply:
Addressed, we can one round of license/security checks after for the first release candidate, nothing major changed in pom.xml since donation.